deploy/us-east-2-dev/secrets/sealed-secrets.yaml

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
These are "sealed secrets"

https://github.com/bitnami-labs/sealed-secrets

Each cluster you'd deploy into has a sealed-secret controller running, which have their own private/public keypair. 

These secrets remain encrypted until they are deployed into the destination cluster. From there, the sealed-secret controller running in the destination cluster will decrypt secrets.

This does make it a bit of a pain, yes. But it lets you store the secrets in git.